src/player.js

```diff
@@ -1248,6 +1249,21 @@ export default class Player extends FakeEventTarget {
     return false;
   }
 
+  _applyTextTrackConfig(): void {
+    if (Utils.Object.getPropertyPath(this._config, 'text.forceCenter')) {
+      this.setTextDisplaySettings({position: 'auto', align: 'center', size: '100'});
+    } else if (Utils.Object.hasPropertyPath(this._config, 'text.textTrackDisplaySetting')) {
+      this.setTextDisplaySettings(this._config.text.textTrackDisplaySetting);
+    }
```

Comment:
I think this will cause object cache issues.
What happens if someone sets `forceCenter` and then sets `textTrackDisplaySetting`? he will need to remeber to set force off.
They can be set seperatly - if force apply default and if specific apply them, at this order.
Otherwise it requires additional call to configure to reset state or at least adding it in config.

Reply:
@OrenMe I think it's a bad configuration, if someone set forceCenter it'll set the `TextDisplaySettings` otherwise his textTrackDisplaySetting config will be applied.
I don't think it's ok to set both, WDYT @OrenMe ?

Reply:
It is always a problem when you have two settings options that impact same internal config, and in these cases it is best to have it as one config.
Obviously this is not possible here - so we remain in finding the least worse.
If you want something to take precedence then you can apply both but call the force after so it applies.
Use cases that are currently a bit problematic are if app wants to change config it needs to set both configs.
Also, if you want to pass both force and other cue settings which force doesn't config you can't do it in one pass also.

Reply:
why would you want to pass both? it's not a weird decision? if you have something special then config from TextDisplaySettings otherwise you can use the default for setting cue on center.
if it'll override from the force after you set specific TextDisplaySettings you'll get a different result then you expect on TextDisplaySettings

Reply:
I do not know the use case as it varies according to specific app requirement.
As it stands now the api is limiting and requires multiple actions to achieve various actions which can be achieved in one call.
The forceCenter is a subset of display settings and can always be applied after it so it takes precedence.
Also if you require setting force and then other attributes that force doesn't affect it won't be possible through studio.